ios/FluentUI/Button/ButtonLegacyTokenSet.swift

```diff
@@ -9,13 +9,21 @@ import UIKit
 
 @objc(MSFButtonLegacyStyle)
 public enum ButtonLegacyStyle: Int, CaseIterable {
```

Comment:
hmmm at this point this should be just same as main and swiftui version should be special?

Reply:
I agree, but that will be addressed in a later PR.

Reply:
Todos? tickets? or ado on ourside?